pkg/http/api.go

```diff
@@ -1906,7 +2033,7 @@ func getMetadataFromRequest(reqCtx *fasthttp.RequestCtx) map[string]string {
 
 func (a *api) onPostStateTransaction(reqCtx *fasthttp.RequestCtx) {
 	if a.stateStores == nil || len(a.stateStores) == 0 {
-		msg := NewErrorResponse("ERR_STATE_STORES_NOT_CONFIGURED", messages.ErrStateStoresNotConfigured)
+		msg := NewErrorResponse("ERR_STATE_STORE_NOT_CONFIGURED", messages.ErrStateStoresNotConfigured)
```

Comment:
Breaking change, not?

Reply:
We might need to document this. Otherwise, LGTM.

Reply:
It's a breaking change in the response text, yes. So far we've considered an API change to be a breaking change if it included a modification for: 1. URL path 2. request/response object 3 .status code change.

If we consider the response text for errors a breaking change then we need to revert this change. Personally I recommend to change and document.

Reply:
I know. Error codes don't have the same treatment as the rest of the API spec. We should rethink this at some point when we start designing the v2 API spec.